extensions/omni_schema/migrate/1_languages.sql

```diff
@@ -5,7 +5,8 @@ create table languages
     language                 name not null,
     extension                name,
     file_processor           name,
-    file_processor_extension name
+    file_processor_extension name,
+    priority                 integer not null default 0
```

Comment:
I like the implementation idea of this table!

Also, yay, I know one more table now! :D